// src/data.ts

export interface Idioma {
    id: number;
    descripcion: string;
}

export interface Palabra {
    id: number;
    palabra: string;
    deletreo: string;
}

export interface Registro {
    id: number;
    idIdioma: number;
    idPalabra: number;
    deletreo: string;
    silabas: number;
    fonetica: string;
}

export const idiomas: Idioma[] = [
    { id: 1, descripcion: "Español" },
    { id: 2, descripcion: "Inglés" },
    { id: 3, descripcion: "Francés" },
    { id: 4, descripcion: "Alemán" },
    { id: 5, descripcion: "Italiano" }
];

export const palabras: Palabra[] = [
    { id: 1, palabra: "casa", deletreo: "ka-sa" },
    { id: 2, palabra: "house", deletreo: "hauz" },
    { id: 3, palabra: "maison", deletreo: "me-zon" },
    { id: 4, palabra: "haus", deletreo: "haus" },
    { id: 5, palabra: "casa", deletreo: "ka-za" }
];

export const registros: Registro[] = [
    { id: 1, idIdioma: 1, idPalabra: 1, deletreo: "ka-sa", silabas: 2, fonetica: "ka-sa" },
    { id: 2, idIdioma: 2, idPalabra: 2, deletreo: "hauz", silabas: 1, fonetica: "hauz" },
    { id: 3, idIdioma: 3, idPalabra: 3, deletreo: "me-zon", silabas: 2, fonetica: "me-zon" },
    { id: 4, idIdioma: 4, idPalabra: 4, deletreo: "haus", silabas: 1, fonetica: "haus" },
    { id: 5, idIdioma: 5, idPalabra: 5, deletreo: "ka-za", silabas: 2, fonetica: "ka-za" }
];
